src/Desktop/Utility/ExtensionMethods.cs

```diff
@@ -206,44 +209,6 @@ private static double GetDPIRate(uint dpi)
             return dpi / 96.0; // 96 is 100% scale. 
         }
 
-        // If a web exception contains one of the following status values, it might be transient.
-        private readonly static HashSet<WebExceptionStatus> TransientWebExceptions = new HashSet<WebExceptionStatus>()
-        {
-            WebExceptionStatus.ConnectionClosed,
-            WebExceptionStatus.Timeout,
-            WebExceptionStatus.RequestCanceled,
-            WebExceptionStatus.NameResolutionFailure
-        };
-
-        /// <summary>
-        /// Attempt to determine if we expect an exception to be transient.
-        /// Will check children of an aggregate exception.
-        /// Based on https://docs.microsoft.com/en-us/azure/architecture/patterns/retry
-        /// </summary>
-        /// <param name="ex">The exception to check</param>
-        /// <returns></returns>
-        public static bool IsTransient(this Exception ex)
```

Comment:
>IsTransient [](start = 27, length = 11)

IsTransient is only used by AIWin code, so it doesn't make sense to have it here--especially since it adds a dependency on System.Net (which is otherwise not needed)